### DISCUSSION TITLE
Forbole Applying for Ecosystem Grant with Forbole X Asset Management Tool

### DISCUSSION THREAD

**Discussion by Zheng-Shilin** [crypto-org-chain/chain-main#525] [Ideas]
Hello Crypto.org community, we at Forbole would like to apply for the Crypto.org ecosystem grant to integrate $CRO into our Forbole X asset management tool. Please see below for our draft proposal. All suggestions and comments are appreciated. Thanks for your time and concern! 

# Contact
Email: info@forbole.com
Team: Forbole
Website: www.forbole.com

# Company/Team Credential 
Forbole is a professional validator for over 25 blockchains and an active contributor in the blockchain space. Forbole has built the reputable open source [Big Dipper](http://bigdipper.live) block explorer, which has been forked over 120 times on GitHub, and received audience from over 140 countries and regions. We recently launched [Desmos Network](http://desmos.network), a sovereign blockchain protocol designed to be the backbone for decentralized social networks apps. The team consists of experienced software developers, infrastructure engineers, seasoned marketers, and creative designers. There are currently 18 full-time employees in the company and we are looking to expand the team in Q3 2021 to over 20 people.

# Project Description 
## Title
Forbole X crypto management tool for CRO

## Introduction 
Forbole X is a multi-token management tool. It will support Cosmos-SDK based chains and is Stargate compatible and IBC enabled. Furthermore, Forbole X will support other ecosystems such as Solana, Flow and potentially Bitcoin and Ethereum. The general version of Forbole X is free to use, while users are able to choose some premium functions, such as generating account reports, with some additional cost. Forbole X comes with a web version and a mobile version. 

## Category
Asset management tool
## How will the project benefit the Crypto.org Chain and its ecosystem
**For $CRO token holders:** 
Forbole X is an easy-to-use yet comprehensive asset management tool. $CRO holders could transfer and stake their tokens conveniently through Forbole X. They would also be able to generate accounting reports if needed, which is a feature missing in other wallets so far. 

**For Crypto.org Chain:**
Forbole X is a multi-coin asset management tool, supporting several Cosmos-SDK based tokens. It can be a gateway to convert other users to become CRO holders. 
Forbole X will be integrated with Big Dipper for Crypto.org too. The user experience can be more seamless combining checking chain status and managing their tokens.  

**For the ecosystem:**
Forbole X can be a valuable asset for the entire ecosystem. It will be a community-built application and can add more variety to the ecosystem, bringing in state-of-the-art blockchain applications development. 

# Timeline of Implementation 
Deliverable | Time (Weeks)
-- | --
UI/UX design for Forbole X - Crypto.org wallet | 2
Setup Crypto.org wallet in Forbole X | 2
Integrate with Big Dipper for Crypto.org | 1
Internal testing | 1
Create a user guide for CRO holders (video and text/pic) | 1
Soft launch | 1
**Total Weeks** | 8

# Budget
Personnel | Number | Rate/week | working time (weeks) | Cost
-- | -- | -- | -- | --
Product/Project Manager | 1 | $2,400 | 8 | $19,200
Engineers | 5 | $2,400 | 6 | $72,000.00
Design | 1 | $2,400 | 2 | $4,800.00
**Total Cost** |   |   |   | $96,000.00

# Project Status
Forbole X web app has been under development for 4 months. We will launch Forbole X web wallet for Desmos as the MVP in May 2021. After that, we will be adding more Cosmos-SDK blockchains into the supporting cohort, including Cosmos Hub, Akash, Band, etc.  We are looking to launch Forbole X mobile in late 2021.

**Comment by mariamar111** [comment]
its good
